.github/workflows/ci.yml

```diff
@@ -37,3 +37,17 @@ jobs:
       uses: actions/checkout@v2
     - name: Run black
       uses: psf/black@stable
+    - name: Run isort
```

Comment:
if you're running more than `black` here, perhaps call this `lint` instead of `black` ?